_ptbr/reply-one.md

```diff
@@ -9,19 +9,19 @@ title: Primeira resposta
+Já faz seis meses desde que saímos de Sarajevo no ônibus sem janelas. Enquanto o ônbus partia eu lhe via acenar e lhe acompanhei com o olhar até se perder à distância. "É bom que o ônibus não tenha janelas", pensei, "caso contrário eu ficara enjoado". Passamos por soldados que ocasionalmente gritavam algo para a gente. Todos vestiam uniformes de cor diferente.
 
-Foi aí que o ônibus quebrou. Bem ao lado de um grupo de soldados. Eles ordenaram que descessemos e revistaram toda a bagagem. Um deles, com chapéu de cowboy e bigode, falava em inglês com outro soldado de boina azul. Ele notou que eu o observava, acariciou minha cabeça e disse-me: "Não se preocupe, tudo vai ficar bem"
+Aí o ônibus quebrou, bem ao lado de um grupo de soldados. Eles ordenaram que todos saíssem do ônbus e começaram a revistar toda a bagagem. Um deles, com chapéu de cowboy e bigode, falava em inglês com outro soldado de boina azul. Ele reparou que eu o observava, deu tapinhas minha cabeça e me disse: "Não se preocupe, tudo vai ficar bem".
 
-Esperamos na multidão até um novo ônibus aparecer. Quando ele finalmente chegou, já estava lotado e tinha janelas. Eu sabia que estaria nele (e que ele me deixaria doente).
+Esperamos todos juntos pelo próximo ônibus. Quando ele finalmente chegou, já estava lotado e tinha janelas. Eu sabia que ia enjoar.
 
-Mamãe abriu caminho no meio da multidão e conseguiu nos levar para o ônibus. Ela nos disse que nos encontraria assim que pegasse nossas mochilas. Eu estava com Stane em um ônibus lotado tentando ver a porta. Quando o ônibus começou a andar, comecei a chorar, apertando os ombros de Stane. A próxima coisa que me lembro foi as pessoas ao meu redor me dando tapinhas nos ombros e dizendo: "Não se assuste, olha lá, sua mãe está aqui..."
+Mamãe abriu caminho no meio da multidão e conseguimos lugar o ônibus. Ela disse que se juntaria a nós assim que pegasse nossas mochilas. Eu fiquei com Stane no um ônibus superlotado, tentando ver a porta. Quando o ônibus começou a andar, comecei a chorar, apertando os ombros de Stane. A próxima coisa que me lembro são as pessoas ao meu redor sendo afetuosas e dizendo: "Não se preocupe, olha lá, sua mãe está aqui."
 
 Eu vomitei naquele ônibus.
 
-Eu me lembro de você dizendo ao se despedir que se juntaria a nós em breve e que enquanto isso, a gente deveria aproveitar Belgrado. Mas não estamos em Belgrado. Vivemos em uma vilinha do interior chamada Divljana, perto de Niš, em um >>resort chamado "Sutjeska". Espero que você venha pra cá em breve.
+Eu me lembro de você dizendo que se juntaria a nós em breve e que enquanto isso, a gente deveria aproveitar Belgrado. Mas não estamos em Belgrado. Estamos em uma pequena vila do interior chamada Divljana, perto de Niš, em um >>resort chamado "Sutjeska". Espero que você venha pra cá em breve.
```

Comment:
```suggestion
Eu me lembro de você dizendo que se juntaria a nós em breve e que enquanto isso, a gente deveria aproveitar Belgrado. Mas não estamos em Belgrado. Estamos em uma pequena vila do interior chamada Divljana, perto de Niš, em um condomínio chamado "Sutjeska". Espero que você venha pra cá em breve.
```

Reply:
I'm not 100% sure on that translation, for we use the word _resort_ in pt-br as well. I usually see "condo" used in en-us to describe what we would call "condomínio". Let's double check with original text.

Reply:
I discussed it with @Stanko and we couldn't find a good word for that. It's like a "conjunto habitacional" built for the working class by the USSR government. Resort sounds like a vacation place here in Brazil.

This is how it looks like:
https://www.juznevesti.com/uploads/assets/2016/12/30/68226/800x600_800x600-divljana-jv.jpg

Reply:
Ok, resort is not the word then! I'll try get some other insights and get back at it!

Reply:
@Stanko we will replace "resort" by "conjunto residencial", that's the most similar expression we have in pt-br. We will add a translation note in another PR so you can decide if you include it or not.

Reply:
Let's keep it like that then 👍